index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Odin_Recipes</title>
+</head>
+<body>
+    <h1>Odin Recipes</h1>
+    <ul>
+        <li><a href="recipes/lasagna.html">lasagna_Recipe</a></li>
+        <li><a href="recipes/Pizza.html">Pizza_Recipe</a></li>
+        <li><a href="recipes/Momo.html">Momo_Recipe</a></li>
+    </ul>
+</body>
+</html>
```

Comment:
All the files should have a new line at the end.

It's a standard to have a blank LF/CRLF as a file terminator. When you will end up working on production-grade applications, linters associated with such applications will end up complaining about such seemingly trivial concerns. It is important therefore that you make it a habit to write clean code from the start.

One [helpful ref.](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3) that gives a gist of why new lines are a must

Pls make a habit of working hand-in-hand with helpful tools like [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) & [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) that come along with the `VSCode` editor & configure them to run automatically on save.

It should save you from the hassle of manually making such or other formatting changes.